graph/ThingGraph.java

```diff
@@ -203,7 +206,7 @@ public ThingVertex convertToReadable(VertexIID.Thing iid) {
         VertexIID.Thing iid = generate(keyGenerator, typeVertex.iid(), typeVertex.properLabel());
         ThingVertex.Write vertex = new ThingVertexImpl.Write.Buffered(this, iid, isInferred);
         thingsByIID.put(iid, vertex);
-        thingsByTypeIID.computeIfAbsent(typeVertex.iid(), t -> new ConcurrentSet<>()).add(vertex);
+        thingsByTypeIID.computeIfAbsent(typeVertex.iid(), t -> new ConcurrentSkipListSet<>()).add(vertex);
```

Comment:
to support using sorted iterators, we must use natively navigable data structures - in this case we use a ConcurrentSkipList to read a sorted iterator from